src/aggregationserver/aggregationbroker/aggregationreplier.cpp

```diff
@@ -133,6 +133,38 @@ AggServer::AggregationReplier::ProcessExperimentStateRequest(const AggServer::Ex
     const int run_id = message.experiment_run_id();
 
     try {
+        const auto& run_results = database_->GetValues(
+            "ExperimentRun",
+            {
+                "JobNum",
+                "to_char((starttime::timestamp), 'YYYY-MM-DD\"T\"HH24:MI:SS.US\"Z\"') AS StartTime",
+                "to_char((endtime::timestamp), 'YYYY-MM-DD\"T\"HH24:MI:SS.US\"Z\"') AS EndTime",
+                "to_char((LastUpdated::timestamp), 'YYYY-MM-DD\"T\"HH24:MI:SS.US\"Z\"') AS LastUpdated"
+            },
+            "ExperimentRunID = " + std::to_string(run_id)
+        );
+        if (run_results.size() == 1) {
```

Comment:
Comment on this might be worthwhile, why one?

Reply:
Addressed in [this commit](https://github.com/cdit-ma/logan/pull/40/commits/dfe7a996577b46a65738668510cc377870085be8)